src/overlays/actors/ovl_En_Mm/z_en_mm.c

```diff
@@ -522,7 +522,7 @@ void EnMm_Update(Actor* thisx, GlobalContext* globalCtx) {
 extern Gfx D_0602CA38[]; // bunny hood dlist from object_link_child. replace with proper symbol later
 
 void EnMm_Draw(Actor* thisx, GlobalContext* globalCtx) {
-    static u64* mouthTextures[] = { gRunningManMouthClosedTex, gRunningManMouthOpenTex };
+    static u64* mouthTextures[] = { gRunningManMouthOpenTex, gRunningManMouthClosedTex };
```

Comment:
I suggest making this be good*

Reply:
We have been using `u64*` for textures everywhere in the repo, so I don't see why change only this one to be `void*`.

Reply:
using void* when possible is something roman has said we may want to do, but yeah it will all be made consistent at the same time

Reply:
I feel like that change is out of scope for this PR, but I could do that change in a future PR :D

Reply:
Cool 👍 a void* pr for all textures could be good, I'm tempted to say that we should just simply make a typedef instead though like SM64 which just has Texture, which I believe in sm64 is actually just a u8, which would make it easy to change the type for textures if we decide to though--anyway that's out of the scope of this pr and we can discuss it some other time, good work on this.